privacy-dashboard/privacy-dashboard-impl/lint-baseline.xml

```diff
@@ -1,6 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="6" by="lint 7.0.4" type="baseline" client="gradle" name="AGP (7.0.4)" variant="all" version="7.0.4">
 
+    <issue
```

Comment:
I've included it here because I don't see how to avoid that lint error.

There's a check in place:
`if (appBuildConfig.sdkInt < VERSION_CODES.Q) return null`

and the method is annotated with `@RequiresApi(VERSION_CODES.Q)`

am I missing something here?

Reply:
Discussed on zoom. Using `@SuppressLint("NewApi")` because lint doesn't capture our check when using `appBuildConfig`.